contracts/Pod.sol

```diff
@@ -623,26 +641,32 @@ contract Pod is
      * @dev Based of the Pod's total token/ticket balance and totalSupply it calculates the pricePerShare.
      */
     function getPricePerShare() external view override returns (uint256) {
-        // Check totalSupply to prevent SafeMath: division by zero
-        if (totalSupply() > 0) {
-            return balance().mul(1e18).div(totalSupply());
+        uint256 _underlying = _calculateUnderlyingTokens(1e18);
+        if (_underlying > 0) {
+            return _underlying;
         } else {
-            return 0;
+            uint256 _decimals = decimals();
+            return 10**_decimals;
```

Comment:
Interesting!  Good idea incorporating the decimals here.

If the _calculateUnderlyingTokens was updated as mentioned previously, I believe you could change this implementation to:

```
  uint256 _decimals = decimals();
  return _calculateUnderlyingTokens(10**_decimals);
```

In fact, I think you have to because the 1e18 constant implies the token has 18 decimal places